src/util/modules.py

```diff
@@ -283,7 +343,17 @@ def rename_module(cr, old, new):
 
 
 def merge_module(cr, old, into, update_dependers=True):
-    """Move all references of module `old` into module `into`"""
+    """
+    Merge a module into another.
+
+    This function moves all references and records from the source module to the
+    destination module. It does not remove any record.
```

Comment:
```suggestion
    destination module. Records from the source module with a conflicting name in the destination module are removed.
```

Note: this can be `warning`, tho.